tests/tacacs/utils.py

```diff
@@ -244,3 +245,37 @@ def remove_all_tacacs_server(duthost):
         tacacs_server = tacacs_server.rstrip()
         if tacacs_server:
             duthost.shell("sudo config tacacs delete %s" % tacacs_server)
+
+
+def check_server_received(ptfhost, data):
```

Comment:
No any code change, for fix the circle import issue, move code from test_accounting.py.